common/autoware_universe_utils/src/math/trigonometry.cpp

```diff
@@ -72,6 +72,19 @@ std::pair<float, float> sin_and_cos(float radian)
   }
 }
 
+// This code is modified from part of OpenCV project
+// (https://github.com/opencv/opencv/blob/4.x/modules/core/src/mathfuncs_core.simd.hpp). It is
+// subject to the license terms in the LICENSE file found in the top-level directory of this
+// distribution and at http://opencv.org/license.html.
```

Comment:
The file is not in the top level directory but in `common/autoware_universe_utils/third_party_licenses/opencv-license.md`